fahrschein-http-apache/src/main/java/org/zalando/fahrschein/http/apache/HttpComponentsRequest.java

```diff
@@ -93,6 +98,10 @@ public final OutputStream getBody() throws IOException {
         assertNotExecuted();
         if (this.bufferedOutput == null) {
             this.bufferedOutput = new ByteArrayOutputStream(1024);
+            if (this.compressEntity) {
+                this.httpRequest.setHeader("Content-Encoding", "gzip");
```

Comment:
Aren't there constants available for that?

Reply:
There's a constant for the header name (org.apache.http.HttpHeaders#CONTENT_ENCODING) not the header value, afaics, but does that make this code more readable?

Reply:
it at least avoids doing Typos?

Reply:
done here: 0881429651b5b2405c4a099864f7767dcf4b9998